packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRow.tsx

```diff
@@ -0,0 +1,183 @@
+import { useCallback, useMemo } from 'react'
+import { twMerge } from 'tailwind-merge'
+import dayjs from 'dayjs'
+import {
+  ArrowTopRightOnSquareIcon,
+  CheckCircleIcon,
+  XCircleIcon
+} from '@heroicons/react/24/outline'
+
+import { DepositStatus, MergedTransaction } from '../../state/app/state'
+import { formatAmount } from '../../util/NumberUtils'
+import { sanitizeTokenSymbol } from '../../util/TokenUtils'
+import { getWagmiChain } from '../../util/wagmi/getWagmiChain'
+import {
+  ChainId,
+  getExplorerUrl,
+  getNetworkName,
+  isNetwork
+} from '../../util/networks'
+import { NetworkImage } from '../common/NetworkImage'
+import { isTxClaimable, isTxFailed, isTxPending } from './helpers'
+import { ExternalLink } from '../common/ExternalLink'
+import { Button } from '../common/Button'
+import { TransactionsTableRowAction } from './TransactionsTableRowAction'
+import { AssetType } from '../../hooks/arbTokenBridge.types'
+
+export function TransactionsTableRow({ tx }: { tx: MergedTransaction }) {
+  const sourceChainId = tx.cctpData?.sourceChainId ?? ChainId.ArbitrumOne
+
+  const { isEthereumMainnetOrTestnet: isSourceChainIdEthereum } =
+    isNetwork(sourceChainId)
+
+  const isClaimableTx = tx.isCctp || tx.isWithdrawal
+
+  const txRelativeTime = dayjs(tx.createdAt).fromNow()
```

Comment:
<img width="959" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/fdfda1f8-670b-435c-9ba8-5c493a8491b6">
feels weird that it doesn't get updated as time passes if you don't reopen it

it wasn't a problem on prod because we're showing a date

but relative time should update (?) at least that's what i expect as a user

Reply:
I will see if we can force trigger it, but with react virtualized it updates only when there's a state update, focus, scroll, etc

Reply:
updated